src/fontra/views/editor/panel-selection-info.js

```diff
@@ -515,7 +522,7 @@ export default class SelectionInfoPanel extends Panel {
           }
         }
       }
-      return "reset axis values";
+      return translate("sidebar.selection-info.component.reset-axis-values.tooltip");
```

Comment:
Here you do include "reset". But likewise, I don't think "tooltip" is needed.

Reply:
done.